Packs/Legacy/Integrations/TruSTAR/TruSTAR.py

```diff
@@ -32,7 +32,7 @@ def translate_indicators(ts_indicators):
     key_context = []
     cve_context = []
     for indicator in ts_indicators:
-        current_indicator = indicator.to_dict(remove_nones=True)
+        current_indicator = indicator if isinstance(indicator, dict) else indicator.to_dict(remove_nones=True)
```

Comment:
Does this solve the bug of ?
![image](https://user-images.githubusercontent.com/37589583/79689652-ec3a4580-825e-11ea-968c-b59e8c08a1a0.png)